install-panel.sh

```diff
@@ -925,7 +906,7 @@ function main {
   required_input user_username "Username for the initial admin account: " "Username cannot be empty"
   required_input user_firstname "First name for the initial admin account: " "Name cannot be empty"
   required_input user_lastname "Last name for the initial admin account: " "Name cannot be empty"
-  password_input user_password "Password for the initial admin account: " "Password cannot be empty"
+  password_input user_password "Password for the initial admin account: "
```

Comment:
Since all the other input functions within the project have two parameters (i.e. a prompt message and a specific error message when the input is empty), I'd like the `password_input` to follow the same project styling.

Reply:
Fair enough I'll change that tomorrow